.browserslistrc

```diff
@@ -0,0 +1 @@
+defaults
```

Comment:
I could swear we had set this up somewhere with different options. Can't track it down though.